playbulb-candle/app.js

```diff
@@ -0,0 +1,111 @@
+var r = g = b = 255; // White by default.
+
+document.querySelector('#connect').addEventListener('click', function() {
+  document.querySelector('#state').classList.add('connecting');
```

Comment:
Not sure what the spec says but maybe you should change the text on the button to say `Connecting...` or disable the button?